demos/crm-sharing/README.md

```diff
@@ -0,0 +1,156 @@
+# Vendia Share Demo - A Real-Time Sharing of CRM Data
+
+[Vendia Share](https://www.vendia.net) is the real-time data cloud for rapidly building applications that securely share data across departments, companies, clouds, and regions.  
+
+## Demo Overview
+
+* __Level__
+    * Intermediate
+* __Highlighted Features__
+    * Fine-grained data access controls
+    * GraphQL
+
+This demo highlights the relative ease that business partners can share account data in a controlled manner.
+
+## Demo Context
+
+In this demo, two joint-venture business partners use Vendia Share to exchange account contact information. Each partner has their own account team managing customer relationships. The joint-venture partner account teams need to coordinate with each other on a regular basis.
+
+Prior to using Vendia Share, the partners struggled to maintain a consistent, shared source of truth of account contact data. The mechanism used was emailing CSVs on a regular, monthly cadence. However, account team structures change often. Since adopting Vendia Share, the partners have been able to reach counterparts in their joint-venture network on the first try, without relying on outdated information.
```

Comment:
```suggestion
Prior to using Vendia Share, the partners struggled to maintain a consistent, shared source of truth of account contact data. They shared account contact information infrequently, periodically emailing CSVs back and forth that often included outdated information.  However, account team structures change much more often.  The result were account teams unable to reach those they wanted to quickly and easily.  Since adopting Vendia Share, the partners have been able to reach counterparts in their joint-venture network on the first try, without relying on outdated information.
```